### DISCUSSION TITLE
Is there a limit to the inbox size?

### DISCUSSION THREAD

**Discussion by yamrzou** [emersion/maddy#644] [Q&A] [answered]
The inbox reached 10000 emails and suddenly recent emails disappeared.

**Comment by foxcpp** [comment, marked as answer]
In 0.7.0 there is indeed a limit, but it is purely a visual one - all messages are still saved when delivered and etc, but only first 10k are displayed.

In https://github.com/foxcpp/maddy/commit/60c25306044684d04ffd2a9926b7250d6faa7082 the limit was increased to 100k. You can build it from source or wait for the next release. In meanwhile you can just move messages to any subfolder.

**Reply by yamrzou** [reply]
Thank you! Yes, once I moved them to another folder, newer ones appeared.
